doc/legacy_mode/linting.md

```diff
@@ -0,0 +1,27 @@
+# Legacy Mode Linting Page
+
+Executing a cargo command makes the extension parse the output of the executed command and show diagnostics.

```

Comment:
\+ `Legacy mode cannot show diagnostics as you type, for that you must use [RLS mode](rls_mode/linting.md).`